libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -21,9 +19,17 @@ export function MediaPlayer({ audioUrl, listenMessage }: MediaPlayerProps) {
         }
 
         audio.play();
+        setIsPlaying(true);
+    };
+
+    const handlePause = () => {
+        audio?.pause();
+        setIsPlaying(false);
     };
 
-    //  const toggle: MouseEventHandler<HTMLButtonElement> = (): void => setPlaying(!playing);
+    const handleAudioNotPlaying = () => {
```

Comment:
It's a really bad idea to introduce our own second layer of state management. The audio player itself has this state (see [MDN  docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playing_event) for the api) and we should hook into that.